.changeset/fluffy-numbers-add.md

```diff
@@ -0,0 +1,5 @@
+---
+"@justeattakeaway/pie-checkbox": minor
+---
+
+[Added] - Default status type
```

Comment:
I don't think this changeset is needed. The other one you have added works - we don't need to treat them like commit messages :)

Reply:
Removed this changeset